components/auth/Login/LoginFormContainer.tsx

```diff
@@ -46,17 +46,6 @@ const LoginFormContainer: React.FC<PropsWithChildren<{}>> = ({ children }) => {
           alert('Success login')
           router.push('/main')
         })
-        .catch((error: any) => {
-          setServerError(error.data.error === 'Unauthorized' ? translate('incorrect_password_or_email') : error.data.error)
-          if (error.status == 'FETCH_ERROR') {
-            alert('Server Error')
-          }
-          if (typeof error.data != 'undefined') {
-            if (error.data.messages === 'invalid password or email') {
-              setServerError('The password or email you entered is incorrect. Please try again!')
-            } else setServerError(error.data.messages)
-          }
-        })
     } catch (error) {
```

Comment:
you left this necessary catch

Reply:
if we use .try, so we have to use .catch. but you're right. removed.